gateway/migrations/0007_user_ids.sql

```diff
@@ -0,0 +1,3 @@
+-- Add new user_id column for projects
+ALTER TABLE users
```

Comment:
I think you meant this:
```suggestion
ALTER TABLE projects
```